Feature/#124 메인페이지 반응형 레이아웃

llddang opened the pull request:
### 관련 이슈
- 이전 이슈인 팀 랭킹 페이지 반응형 부터 봐주세요! #117 
- close #124 

### 작업 요약
- 화면의 크기에 따라 글씨의 크기, 작물의 크기, 로그인 버튼 크기가 변경되도록 했습니다.
- 화면의 너비가 768px 보다 작을 때는 작물이 사라지도록 했습니다.

### 리뷰 요구 사항
- 제 노트북에서 확인했는데요, 여러분 노트북에서도 괜찮은지 궁금합니다.
- GoogleLoginButton 컴포넌트에서 `const isLogin = false;` 했을 때에도 화면 괜찮은지 확인해주시면 감사하겠습니다.
- 리뷰 예상 시간 : `5분`
- 화면의 너비가 768px ~ 992px 사이일 때 화면인데요. 둘 중 어느 것이 나은지 궁금합니다.
   한 개는 글씨를 세로 가운데 정렬한 것이고요, 나머지 한개는 baseline에 정렬한 다음 pt="100px"을 주었습니다. <img width="40%" alt="스크린샷 2024-03-11 오후 11 56 36" src="https://github.com/BDD-CLUB/01-doo-re-front/assets/77055208/645daaf4-5c48-4402-b06f-6b1a8487827d"> <img width="40%" alt="스크린샷 2024-03-11 오후 11 56 23" src="https://github.com/BDD-CLUB/01-doo-re-front/assets/77055208/a8e50d6f-0d0f-4c5c-afd1-5130c58efe94">

llddang committed:
design: 반응형을 위한 font style 적용

#110

llddang committed:
feat: 사이드바의 CaregoryButton의 isTeam, isTeamMatch props의 기본값을 false로 주기.

#110

llddang committed:
chore: 반응형에 따라서 크기가 변경되도록 icon button size 재설정

#112

llddang committed:
Merge branch 'feature/#112-반응형_버튼_크기_지정' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#110-사이드바_반응형_수정

llddang committed:
refactor: 사이드바의 햄버거 크기 설정을 좀 더 보기 좋게 변경함.

#110

llddang committed:
design: side bar에서 width 속성 삭제

- width이 고정이 아닌, fit-content로 사용됨.
#110

llddang committed:
Merge branch 'develop' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#110-사이드바_반응형_수정

# Conflicts:
#	src/theme/components/button.ts

pipisebastian committed:
design: 사이드바 모바일 반응형 구현

#110

pipisebastian committed:
design: 모바일 사이드바 열려있을때 내부 콘텐츠 밀리도록 수정
#110

llddang committed:
feat: 화면의 크기에 따른 반응형 조절 방식 변경

- 110

llddang committed:
design: 팀 랭킹페이지의 소개글 반응형 구현

#117

llddang committed:
refactor: 팀 랭킹 페이지의 페이지 소개글에서 사용되지 않는 css 삭제

#117

pipisebastian committed:
design: 사이드바 초기상태 닫히도록 수정 및 배경색 추가
#110

llddang committed:
design: 팀 랭킹 페이지에서 페이지 소개글의 border 삭제

- 레이아웃 확인하기 위해서 추가했던 border 속성 삭제
- #117

llddang committed:
chore: react 컴포넌트 - swiper 설치

- #117

llddang committed:
design: 팀 랭킹페이지의 소개글 textStyles 값 변경

- 페이지에서 보기에 글씨가 너무 커보여서 바꾸었습니다.
- #117

llddang committed:
chore: 팀 카드의 props 추가 (idx)

- swiper에서 index로 slide를 넘겨주는데, pagination 과 연결시켜주기 위해 추가했습니다.
- #117

llddang committed:
design: 팀 랭킹페이지의 slide 부분 수정

- 기존에는 제가 작성한 slide 를 넣어는데요, swiper 라이브러리를 알게되어 변경했습니다.
- #117

llddang committed:
refactor: 주석 삭제

- #117

llddang committed:
Merge branch 'feature/#110-사이드바_반응형_수정' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#117-팀_랭킹_페이지_반응형_레이아웃

llddang committed:
fix: 사이드바 zindex 범위 조정

- 사이드 바 전체에 zIndex = 99로 설정하니, 뒤에 있는 요소들을 선택할 수가 없음.
- 그래서 화면의 width이 768px보다 작고 사이드바가 펼쳐졌을 때만, zIndex = 2로 설정.
- #110

llddang committed:
refactor: 사이드바 배경과, 사이드바 통합.

- #110

llddang committed:
Merge branch 'feature/#110-사이드바_반응형_수정' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#117-팀_랭킹_페이지_반응형_레이아웃

llddang committed:
feat: 팀 랭킹 페이지 반응형 구현

- #117

llddang committed:
Merge branch 'feature/#117-팀_랭킹_페이지_반응형_레이아웃' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#124-메인베이지_반응형_레이아웃

# Conflicts:
#	src/app/layout.tsx

llddang committed:
design: 화면크기에 따른 글씨 크기 바꿔주기.

- 처음에 화면에 대입하지 않고 설정하여, 어색한 부분 다시 설정해주었습니다.
- #124

llddang committed:
design: 메인페이지의 로그인 버튼 화면 크기에 따라 크기 변화

- 변화하지 않고 있으니, 너무 띄는 것 같아서 조절해주었습니다.
- #124

llddang committed:
design: 메인페이지에서 작물의 크기가 화면의 크기에 따라 변화하도록 수정

- #124

llddang committed:
design: 메인페이지 반응형 반영

- 화면의 너비가 768px보다 작을 때는 작물이 사라지도록 했습니다.
- 반응형이도록 만든 textStyle을 text요소에 적용시켰습니다.
- #124

llddang committed:
design: 작물이 사라지게 하는 화면의 너비 변경

- 화면의 너비가 768px일 때 사라지는 줄 알았는데, 992px 일때 사라지고 있어서 수정했습니다.
- #124

jasper200207 reviewed (approved):
👍

yeonddori reviewed (approved):
로그인/로그아웃 상태 모두 좋네요! 수고하셨습니다👍
저는 개인적으로 pt="200px" 정도가 마음에 드네요..ㅎㅎ
하지만 미적 감각이 없는 걸 알기에 다른 분들의 의견에 따르겠습니다~

llddang commented:
> 로그인/로그아웃 상태 모두 좋네요! 수고하셨습니다👍 저는 개인적으로 pt="200px" 정도가 마음에 드네요..ㅎㅎ 하지만 미적 감각이 없는 걸 알기에 다른 분들의 의견에 따르겠습니다~

굿굿!!
아뇨 저도 아 두개의 중간이 좋을 것 같은데 라고 생각했습니다!!

llddang committed:
design: 가로 스크롤바 없애기

- overflow: scroll 로 설정하면 안의 내용에 상관없이 가로, 세로 스크롤바가 무조건 생성됩니다.
- overflow: auto 로 설정하면 안의 내용 크기에 따라 스크롤바가 생깁니다.
- #117

llddang committed:
design: swiper가 부모요소의 크기 넘지 않도록 설정

- #117

llddang committed:
fix: 화면의 세로 길이를 늘렸을 때, 윗부분 세션이 보이는 에러 수정

- #117

mun-kyeong reviewed (approved):
좋습니다!!
<img src="https://github.com/BDD-CLUB/01-doo-re-front/assets/96461334/02ac98b5-a926-4787-938d-a82f6126a50d" width="50%">
저는 개인적으로 이거 앞에 이미지가 마음에 드네욤 :)

llddang committed:
Merge branch 'feature/#117-팀_랭킹_페이지_반응형_레이아웃' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#124-메인페이지_반응형_레이아웃

llddang commented:
> 좋습니다!! <img alt="" width="50%" src="https://private-user-images.githubusercontent.com/96461334/312816089-02ac98b5-a926-4787-938d-a82f6126a50d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA0MTkyODIsIm5iZiI6MTcxMDQxODk4MiwicGF0aCI6Ii85NjQ2MTMzNC8zMTI4MTYwODktMDJhYzk4YjUtYTkyNi00Nzg3LTkzOGQtYTgyZjYxMjZhNTBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE0VDEyMjMwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3MjYzNzU5MjA1NTE0ZGE1NDEwYTM0NDE1MWJjMDYyNzRmMzU0ZjA2ODMyODUzYmNhMDU5NzI3YTdlNWM3OWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M-ydQkGzeoVFlkBSZk9Me4cESQHJWD5v5vWFg68h4oI"> 저는 개인적으로 이거 앞에 이미지가 마음에 드네욤 :)

가운데 정렬로 가겠습니다!

llddang committed:
design: 플랫폼 소개글 세로 중앙 정렬로 수정

- #124

llddang committed:
Merge branch 'develop' of https://github.com/BDD-CLUB/01-doo-re-front into feature/#124-메인페이지_반응형_레이아웃

# Conflicts:
#	src/app/page.tsx

pipisebastian reviewed (approved):
구시에용~
